feedbacks/urls.py

```diff
@@ -1,5 +1,6 @@
 from django.urls import path
-from .views import *
+from .views import FeedbackListView, FeedbackCreateView, FeedbackDeleteView, \
```

Comment:
According to [pep-8](https://www.python.org/dev/peps/pep-0008/#imports) a blank line before `.views` is required